test/linalg.jl

```diff
@@ -229,4 +227,8 @@ using Test
             end
         end
     end  # SVD
+    @testset "Left division" begin
+        @test b1 \ a ≈ inv(b1) * a
+        @test_throws ArgumentError BlockDiagonal([A, B]) \ rand(rng, 2N + N1 + N2)
```

Comment:
sorry that existing tests here don't set a good precedent for this: would it be easy enough to define our own variables for the `testset`, rather than pre-defined ones? just in reviewing these changes i notice how hard to having to go up hundreds of lines of code to check what these variables are.

Reply:
Yeah I was trying to follow the style of the rest of the file. But I am happy to define the variables locally